app/src/Containers/SearchContainer.tsx

```diff
@@ -294,7 +297,33 @@ const SearchContainer = () => {
     })
   }
 
-  const searchByArray = (
+  const exactSearchByArray = (
+    searchQueries: string[],
+    searchField: 'client' | 'description' | 'name' | 'role' | 'skills' | 'sector',
+  ): Projects => {
+    let results = [] as Projects
+
+    if (projects) {
+      results = projects.filter(
+        project => {
+          const anySearchQueriesMatching = searchQueries.some(searchQuery => {
```

Comment:
Just a style thing: booleans usually have names starting with `is`, so a name like `isMatch` would be a helpful indicator that it's a boolean.

Or alternatively, since you're not doing anything with the boolean other than returning it, you could just return it straight away without giving it a name first: return searchQueries.some(...)

Reply:
Good call Rosie, thanks!